test/index.js

```diff
@@ -15,24 +16,26 @@ describe('inquirer-npm-name', function () {
     this.inquirer = {
       prompt: sinon.stub()
     };
+
   });
 
   it('only ask name if name is free', function (done) {
-    this.inquirer.prompt.callsArgWith(1, {name: 'foo'});
 
-    askName(this.conf, this.inquirer, function (val) {
-      assert.equal(val, 'foo');
+    this.inquirer.prompt.returns(Promise.resolve({name:'foo'}));
+
+    return askName(this.conf, this.inquirer).then(function (props) {
+      assert.equal(props.name, 'foo');
       done();
```

Comment:
These tests are already on mocha.

You need to remove the `done` from the test function arguments list too.